website/docs/language/stacks/deploy/link-stacks.mdx

```diff
@@ -0,0 +1,114 @@
+
+For example, you can add a `publish_output` block for the `vpc_id`  in your upstream Stack’s deployment configuration.
+
+<CodeBlockConfig filename="network.tfdeploy.hcl">
+
+```hcl
+# Networking Stack deployment configuration
+
+publish_output "vpc_id" {
+  description = "The networking Stack's VPC's ID."
+  # You can directly reference a deployment's values with the
+  # deployment.deployment_name syntax
+  value       = deployment.network.vpc_id
+}
+```
+
+</CodeBlockConfig>
+
+After applying this configuration, any Stack in the same project can now reference this `vpc_id` output by declaring an `upstream_input` block. Learn more about the [`publish_output` block](/terraform/language/stacks/reference/tfdeploy#publish_output-block-configuration).
+
+## Use an upstream Stack’s inputs
+
+Declare an `upstream_input` block in your Stack’s deployment configuration to read values from another Stack's `publish_output` block. Adding an `upstream_input` block creates a dependency on the upstream Stack.
+
+For example, if you want to use the output `vpc_id` from an upstream Stack in the same project, declare an `upstream_input` block in your deployment configuration. 
+
+<CodeBlockConfig filename="application.tfdeploy.hcl">
+
+```hcl
+# Application Stack deployment configuration
+
+upstream_input "networking_stack" {
+  type   = "Stack"
+  source = "app.terraform.io/hashicorp/Default Project/networking-stack"
+}
+
+deployment "application" {
+  inputs = {
+    # This Stack depends on the networking Stack for this value
+    vpc_id = upstream_input.network_stack.vpc_id
+  }
+}
+```
+
+</CodeBlockConfig>
+
+After pushing your Stack's configuration into HCP Terraform, HCP Terraform searches for the most recently published snapshot of the upstream Stack your configuration references. If no snapshot exists, the downstream Stack's run fails. 
+
+If HCP Terraform finds a published snapshot for your referenced upstream Stack, then all of that Stack's outputs are available to this downstream Stack. Add `upstream_input` blocks for every upstream Stack you want to reference. Learn more about the [`upstream_input` block](/terraform/language/stacks/reference/tfdeploy#upstream_input-block-configuration).
+
+To stop depending on an upstream Stack’s outputs, do the following in your downstream Stack's deployment configuration:
+
+- Remove the upstream Stack's `upstream_input` block  
+- Remove any references to the upstream Stack's outputs
+- Push your configuration changes to HCP Terraform and apply the new configuration
+
+## Trigger runs when output values change
+
+If an upstream Stack's published output values change, HCP Terraform automatically triggers runs for any downstream Stacks that rely on those outputs. 
+
+For example, if your upstream networking Stack’s output changes, HCP Terraform triggers a new plan for the downstream Stacks that reference that output.
```

Comment:
```suggestion
In the following example, the `application` deployment is dependent on the upstream source. As a result, it receives new data when the upstream source changes and triggers a new HCP Terraform run.
```

Reply:
Made shorter and added the context below the example.